pkg/ibmvpc/explainability_test.go

```diff
@@ -34,77 +34,77 @@ var explainTests = []*vpcGeneralTest{
 	{
 		name:        "VsiToVsi1",
 		inputConfig: "sg_testing1_new",
-		ESrc:        "vsi2-ky[10.240.20.4]",
-		EDst:        "vsi3b-ky[10.240.30.4]",
+		ESrc:        "vsi2-ky",
```

Comment:
is there a test with src or dst as CRN of a VSI?

Reply:
No, I was not sure how useful the crn will be.
The point here is that VSI name may not be unique - so why use crn for UID and not the actual id?
Plus, currently our "crn" is not a real crn thus a test might not be useful, as it will not test a real crn which has certain non standard characters.

Anyways, I've added a test. Should I open an issue (for using id instead of crn)?

Reply:
The UID for any resource in the model is the CRN, except for few resources that do not have CRN, for which ID is used. 
(both CRN and ID are unique). 
It was decided to use CRN when possible when started modeling. (`parser.go` assigns those values).
I don't think there is an issue in testing a fake crn, just need to make sure that it matches the right field from the config object.

Reply:
I've modified the "crn" test to have an exact "crn" format, which includes (of course) changing the relevant test file.

However, my point is that it is very unlikely anyone will actually use the `crn` as input. This is not the case for the `id`.  Why not use the id as UID?

Reply:
It was decided to use crn and not id as Karen suggested.
@kyorav what do you think?